### ISSUE TITLE
Element Picker for Per-Site Exclusion Rules

### ISSUE THREAD

**Issue by lhl** [alexanderby/darkreader#48]
Hi, just discovered Dark Reader, and it's great. I didn't see this in the issues, but I'd be interested seeing what the interest/feelings are on support for more extensive per-site customization/overrides.

Ideally, it'd be nice to be able to adjust some of the rules (like contrast) on a per-site level, and on a per-element basis (using an element picker) - is anyone using Dark Reader w/ Stylish? Maybe there's easy ways to carve out exclusions that way.

Otherwise I'd be interesting in forking and doing some work on a version that has some of this functionality, maybe also making the ignore/darksites JSON file be something that could auto-update/be customized - I see some pull requests there? Just wanted to see what you think before I get started @alexanderby 

What's the darkreader license btw? There are some plugins that already have some functionality, but uBlock for example is GPLv3, and there might be other code that'd be useful that's MIT licensed.

**Comment by brodock** [comment]
Being able to pick elements (or adding the XPATH or selector rules on a site) would be super useful. There are elements that are already "darker" and I don't want it to have inverted colors (like the gitlab.com code highlighted blocks).

**Comment by alexanderby** [comment]
Maybe Chrome Dev Tools picker it a better choice, because more information is needed to determine which selector suits better. Here is an updated doc for modifying per-site fixes https://github.com/alexanderby/darkreader#how-to-contribute

**Comment by elypter** [comment]
why not copy the one from ublock origin?

**Comment by Gusted** [comment]
> why not copy the one from ublock origin?

License and we will be publicly shamed for stealing code :).

**Comment by elypter** [comment]
cant they just switch from mit to gpl?

**Comment by Gusted** [comment]
> cant they just switch from mit to gpl?

That is not how it work...

**Comment by DatHypnoboi** [comment]
I desperately want this to be added

**Comment by ggplayer0** [comment]
Honestly, this not being implemented in some form is the only thing keeping me from using this extension on a regular basis. Navigating through the dev tools just to fix one part of a page is a frustrating experience. 
Would there really be a licensing issue with using the selector from UBlock Origin? I'm _not_ a legal expert, but it seems like the licenses are compatible.

**Comment by Shineblind** [comment]
yeah this is something that would be super useful, sometimes eg. if images are black, element picker + a hotkey for it would be amazing
